xapian-applications/omega/templates/query

```diff
@@ -68,7 +70,9 @@ $if{$opt{topterms},
  <br><noscript><input type=hidden name=ADD value=1></noscript>
  </div>
 }
+</fieldset>
 <br>
+<section>
```

Comment:
> Do not use the <section> element as a generic container; this is what <div> is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.

(from the [MDN description of <section>](https://developer.mozilla.org/en/docs/Web/HTML/Element/section))

I'm not sure this counts, but the way to think about it is: what are the names of the different parts of the page? Give them all headings that say what they're doing (when we add default CSS, we can throw them off the page using `position: absolute; left: -9999px`). That should make it clear when to use sections (and whether they need `id` or `class` to distinguish each of them).